thelounge/conf/public/config.js

```diff
@@ -0,0 +1,458 @@
+	//   this limit will be prompted with an error message in their browser. A value of
+	//   `-1` disables the file size limit and allows files of any size. **Use at
+	//   your own risk.** This value is set to `10240` kilobytes by default.
+	// - `baseUrl`: If you want change the URL where uploaded files are accessed,
+	//   you can set this option to `"https://example.com/folder/"` and the final URL
+	//   would look like `"https://example.com/folder/aabbccddeeff1234/name.png"`.
+	//   If you use this option, you must have a reverse proxy configured,
+	//   to correctly proxy the uploads URLs back to The Lounge.
+	//   This value is set to `null` by default.
+	fileUpload: {
+		enable: false,
+		maxFileSize: 10240,
+		baseUrl: null,
+	},
+
+	// ### `transports`
+	//
+	// Set `socket.io` transports.
+	//
+	// This value is set to `["polling", "websocket"]` by default.
+	transports: ["polling", "websocket"],
+
+	// ### `leaveMessage`
+	//
+	// Set users' default `quit` and `part` messages if they are not providing
+	// one.
+	//
+	// This value is set to `"The Lounge - https://thelounge.chat"` by
+	// default.
+	leaveMessage: "*poof*",
+
+	// ## Default network
+
+	// ### `defaults`
+	//
+	// Specifies default network information that will be used as placeholder
+	// values in the *Connect* window.
+	//
+	// The available keys for the `defaults` object are:
+	//
+	// - `name`: Name to display in the channel list of The Lounge. This value is
+	//   not forwarded to the IRC network.
+	// - `host`: IP address or hostname of the IRC server.
+	// - `port`: Usually 6667 for unencrypted connections and 6697 for
+	//   connections encrypted with TLS.
+	// - `password`: Connection password. If the server supports SASL capability,
+	//   then this password will be used in SASL authentication.
+	// - `tls`: Enable TLS connections
+	// - `rejectUnauthorized`: Whether the server certificate should be verified
+	//   against the list of supplied Certificate Authorities (CAs) by your
+	//   Node.js installation.
+	// - `nick`: Nick name. Percent signs (`%`) will be replaced by random
+	//   numbers from 0 to 9. For example, `Guest%%%` may become `Guest123`.
+	// - `username`: User name.
+	// - `realname`: Real name.
+	// - `join`: Comma-separated list of channels to auto-join once connected.
+	//
+	// This value is set to connect to the official channel of The Lounge on
+	// Freenode by default:
+	//
+	// ```js
+	// defaults: {
+	//   name: "Freenode",
+	//   host: "chat.freenode.net",
+	//   port: 6697,
+	//   password: "",
+	//   tls: true,
+	//   rejectUnauthorized: true,
+	//   nick: "thelounge%%",
+	//   username: "thelounge",
+	//   realname: "The Lounge User",
+	//   join: "#thelounge"
+	// }
+	// ```
+	defaults: {
+		name: "Network",
+		host: "your.website.tld",
+		port: 6697,
+		password: "",
+		tls: true,
+		rejectUnauthorized: true,
+		nick: "Anonymous",
+		username: "Anonymous",
+		realname: "Anonymous",
+		join: "#main",
+	},
```

Comment:
Shouldn't this be `name: "FishNet"`, `host: "irc.wetfish.net"`, and `join: "#wetfish"`?

Reply:
It should, and it is now. :)